acp/wpn_acp_module.php

```diff
@@ -117,6 +117,12 @@ public function save_settings()
 			'wpn_webpush_vapid_private'=> ['validate' => 'string:25:255', 'lang' => 'WEBPUSH_VAPID_PRIVATE'],
```

Comment:
Can you just set this to `'validate' => 'password:25:255'` and that will handle everything?

Reply:
That would be handy but there's no such a thing in config vars validator. `build_cfg_template` can handle displaying password as asterixes but preventing its updating and validation is still handled within dedicated xCP module.